k8s.gcr.io/lib.sh

```diff
@@ -191,6 +191,34 @@ function ensure_gcs_bucket() {
     gsutil iam ch allUsers:objectViewer "${bucket}"
 }
 
+# Sets the web policy on the bucket, including a default index.html page
+# $1: The GCP project
+# $2: The bucket
+function ensure_gcs_web_policy() {
+    if [ $# -lt 2 -o -z "$1" -o -z "$2" ]; then
+        echo "ensure_gcs_web_policy(project, bucket) requires 2 arguments" >&2
+        return 1
+    fi
+    project="$1"
```

Comment:
you never use this?

Reply:
Yeah, and here you can see that I am inconsistent because for copy I didn't pass the project.

I can rationalize them all if you want: Pass the (unused) project or not?